### DISCUSSION TITLE
How to remove all traces of Feishin in MacOS?

### DISCUSSION THREAD

**Discussion by ghost** [jeffvli/feishin#274] [Q&A]
Hey,
I have a weird problem with Feishin where if I press play, it won't play the acutal music. It's on 0:00 all the time. Ultrasonic on my Android phone works, but here on MacOS it doesn't. So it's not a problem with my server - Navidrome.

I suspect this may be caused by some lingering files from previous versions (0.1) itd that cause 0.4 to behave weirdly. When I remove the /Applications/Feishin.app folder and reinstall, my saved server are still there. Which means I'm not really working on a fully clean install.

Which files should I remove to get a completely fresh install?

**Comment by ghost** [comment]
The URL of my navidrome instance changed and the app starts, loads for a long time, then displays a message "No issues found".  

Removing the app and reinstalling does not change the behaviour so I agree there are some configurations still in the system after deinstalling.  

Also there really is no option to change the server address from this point:
<img width="1435" alt="Screenshot 2024-04-29 at 14 03 44" src="https://github.com/jeffvli/feishin/assets/10931525/69f85e5e-1e38-45fa-8fff-ba98bb943021">


### My Solution

Using `sudo mdfind "feishin"` I found the folder:
`/Users/<User>/Library/Application Support/feishin`, removing the folder and restarting the app allowed me to readd the server.   

I guess it goes without saying to delete it at your own risk but maybe this helps you finding other artifacts of previous versions.

**Reply by kgarner7** [reply]
This is resolved by https://github.com/jeffvli/feishin/commit/44fcc338257224621b3582e159d4e85150844e5f, so you can either build it yourself (or pick any of the builds from https://github.com/jeffvli/feishin/actions/runs/8888771003)
